src/pal/src/file/directory.cpp

```diff
@@ -581,7 +573,6 @@ CreateDirectoryA(
         }
     }
 
-    // Check the constraint for the real path length (should be < MAX_LONGPATH).
 
     // Get an absolute path.
     if (UnixPathName[0] == '/')
```

Comment:
I wonder why we convert relative path into absolute path before calling mkdir. Unix mkdir function can handle relative path. My guess is the only reason for getting the absolute path is in order to check its length. If that's the case, we should no get rid of that conversion.